.fuel-core/configs/chainConfig.json

```diff
@@ -129,6 +129,12 @@
           "wqmm": 10,
           "xor": 2,
           "xori": 2,
+          "alloc": {
+            "LightOperation": {
+              "base": "2",
+              "unitsPerGas": "214"
+            }
+          },
```

Comment:
Why are we using `alloc` here? It seems that the added gas cost instruction was `alocDependentCost`.

Also, it appears that this gas instruction is not set on the dev/test nets
<img width="743" alt="Screenshot 2024-06-12 at 07 11 49" src="https://github.com/FuelLabs/fuels-ts/assets/30977845/499439a1-4287-4b01-a498-8c0c61396296">

Reply:
UPDATE:

`alocDependentCost` it is set on the `devnet`, but not on the `testnet`, very likely because it was not updated yet.